cmd/celestia/main.go

```diff
@@ -5,9 +5,32 @@ import (
 	"os"
 
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"github.com/celestiaorg/celestia-node/cmd/celestia/bridge"
+	"github.com/celestiaorg/celestia-node/cmd/celestia/full"
+	"github.com/celestiaorg/celestia-node/cmd/celestia/light"
```

Comment:
The reason they were put into subpkgs is because we cannot import from the main. However, I don't think it  a sub pkg per node type and we have `cmd` where all the reusables are kept for now, so let's move node type cmds there.

Reply:
It can be somthing like `cmd.NewBridge` etc,

Reply:
Moving to `cmd/celestia/node.go` and renaming to `NewBridge/NewLight/NewFull` 👍🏽